package.json

```diff
@@ -2,15 +2,15 @@
   "name": "session-desktop",
   "productName": "Session",
   "description": "Private messaging from your desktop",
-  "version": "1.14.2",
+  "version": "1.15.0",
   "license": "GPL-3.0",
   "author": {
     "name": "Oxen Labs",
     "email": "team@oxen.io"
```

Comment:
Does this not need to be updated to the foundation?

Reply:
you mean the email team@oxen.io?

Reply:
Yes and and the Oxen Labs

Reply:
this should be resolved now. Will let you do it.